frontend/types/src/components/EndoAI/VideoAnnotation.vue.d.ts

```
@@ -1,6 +1,29 @@
-import 'filepond/dist/filepond.min.css';
-declare const _default: import("vue").DefineComponent<{}, {}, {}, {}, {}, import("vue").ComponentOptionsMixin, import("vue").ComponentOptionsMixin, {}, string, import("vue").PublicProps, Readonly<{}> & Readonly<{}>, {}, {}, {}, {}, string, import("vue").ComponentProvideOptions, true, {
-    videoRef: HTMLVideoElement;
-    pond: unknown;
-}, HTMLDivElement>;
+declare const _default: import("vue").DefineComponent<{}, {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)